src/views/auth/TheLogin.vue

```diff
@@ -0,0 +1,154 @@
+<script setup lang="ts">
+// import { storeToRefs } from 'pinia'
+import { useAuthStore } from '../../stores/auth.ts'
+import { ref } from 'vue'
+import { useRouter } from 'vue-router'
+
+import loginImg from '../../assets/identity.svg'
```

Comment:
Why don't we create an svg folder img inside assets and keep all the svgs over there.